app/models/chapter.rb

```diff
@@ -5,6 +5,7 @@ class Chapter < ApplicationRecord
   include HtmlCleaner
   include WorkChapterCountCaching
   include Creatable
+  include CreatorshipTests
```

Comment:
CreatorshipValidations? The current name makes it sound like it's for testing only.